src/app/OperationalSessionSetup.h

```diff
@@ -109,6 +109,29 @@ class OperationalDeviceProxy : public DeviceProxy
     ScopedNodeId mPeerScopedNodeId;
 };
 
+enum class CaseConnectionState : uint8_t
```

Comment:
It seems we are using the term 'Connection' interchangeably with 'Session Establishment'. The API is FindOrEstablishSession() but herein we are calling it a connection. This would, especially, muddy the waters when we have TCP as the underlying transport. Would have preferred if it were named as Session. Changing it everywhere would have a lot of touch points though.

Reply:
I agree with this renaming, but it changed the public API and we need to update all references. Track this change in https://github.com/project-chip/connectedhomeip/issues/29303